pom.xml

```diff
@@ -516,6 +516,35 @@
       </build>
     </profile>
 
+    <profile>
+      <id>clean-node-modules</id>
+      <activation>
+        <property>
+          <name>skipNodeClean</name>
```

Comment:
Can you try removing the value tag and just leave <name>!skipNodeClean</name>? This should make it so the presence of a -DskipNodeClean will not trigger the profile.

Reply:
It works like that already, but I'll try removing it.